docs/book/src/cronjob-tutorial/testdata/project/main.go

```diff
@@ -76,17 +76,19 @@ func main() {
 		Scheme:             scheme,
 		MetricsBindAddress: metricsAddr,
 		LeaderElection:     enableLeaderElection,
+		Port:               9443,
```

Comment:
I don't understand this change.

Reply:
The docs is outdate. See: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v2/main.go#L63

Reply:
Right, it is not related to the documentation but updating the docs.

I would say that there are way more places where this happens and that it may be a good decission to fully update it in a separate PR. And by fully updating I mean following the tutorial step by step and using the files generated as the base for the book.

Reply:
HI @Adirio, 

Did you find other places?  I could not find them. Please, let me know if you check any other place that is missing the port.

Reply:
I did not find any other `Port` missing, I was talking in general that the files used for the book was generated with an older version ok kubebuilder, and thus the files may have changed. Some of the changes in the new versions have been also placed in the book like the `schema` you set in a previous PR, but probably others were not updated in the docs files.

Reply:
HI @Adirio, 

If you check the testdata, or create a new project, you can see that the PORT is there as in other places. So, this doc is outdated. Because of this the Port was added here.  


,

Reply:
I can not work on it right now, I'm working on the scaffold enhancements, plus I'm on a windows machine when certain commands from kubebuilder do not work so a linux user should tackle this.

Reply:
So, I tracked an issue for that here: https://github.com/kubernetes-sigs/kubebuilder/issues/1251. 

It is not part of the scope of this PR as you said above and then I am closing these comments as solved.